tests/integration/test_put_along_axis.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+from test_tools.generators import mk_seq_array
+
+import cunumeric as num
+from legate.core import LEGATE_MAX_DIM
+
+
+def test_3d():
+
+    x = mk_seq_array(np, (256, 256, 100))
+    x_num = mk_seq_array(num, (256, 256, 100))
+
+    indices = mk_seq_array(np, (256, 256, 10)) % 100
+    indices_num = num.array(indices)
+
+    res = np.put_along_axis(x, indices, -10, -1)
```

Comment:
Here also compare `x` after the updates, not the return value of `put_along_axis`.